subprojects/docs/src/docs/userguide/what_is_gradle.adoc

```diff
@@ -12,133 +12,112 @@
+It also makes it easy to run Gradle on different platforms.
 Conventions::
-Gradle takes a leaf out of Maven's book and makes common types of projects — such as Java projects — easy to build by implementing conventions. Apply the appropriate plugins and you can easily end up with slim build scripts for many projects. But these conventions don't limit you: Gradle allows you to override them, add your own tasks, and make many other customizations to your convention-based builds.
+Gradle makes common types of projects easy to build through conventions. Plugins set sensible defaults to keep build scripts minimal.
+But these conventions don't limit you: you can configure settings, add your own tasks, and make many other customizations in your builds.
 Extensibility::
-You can readily extend Gradle to provide your own task types or even build model. See the Android build support for an example of this: it adds many new build concepts such as flavors and build types.
+You can readily extend Gradle to provide your own build logic with custom tasks and plugins.
+See Android builds for an example: they add many new build concepts such as flavors and build types.
 IDE support::
-Several major IDEs allow you to import Gradle builds and interact with them: Android Studio, IntelliJ IDEA, Eclipse, and NetBeans. Gradle also has support for generating the solution files required to load a project into Visual Studio.
+Several major IDEs enable imports and interaction with Gradle builds, including Android Studio, IntelliJ IDEA, Eclipse, and NetBeans.
+Gradle can also provides generate the solution files required to load a project into Visual Studio.
 Insight::
-https://scans.gradle.com/[Build scans] provide extensive information about a build run that you can use to identify build issues. They are particularly good at helping you to identify problems with a build's performance. You can also share build scans with others, which is particularly useful if you need to ask for advice in fixing an issue with the build.
-
-[[five_things]]
-== Five things you need to know about Gradle
-
-Gradle is a flexible and powerful build tool that can easily feel intimidating when you first start. However, understanding the following core principles will make Gradle much more approachable and you will become adept with the tool before you know it.
+https://scans.gradle.com/[Build scans] provide extensive information about a build that you can use to identify issues.
+You can use build scans to identify problems with a build's performance, and even share them for debugging help.
 
-=== 1. Gradle is a general-purpose build tool
+== Concepts
 
-Gradle allows you to build any software, because it makes few assumptions about what you're trying to build or how it should be done. The most notable restriction is that dependency management currently only supports Maven- and Ivy-compatible repositories and the filesystem.
```

Comment:
I feel like "Gradle allows you to build any software, because it makes few assumptions about what you're trying to build or how it should be done." was an important point that should remain somewhere.

Reply:
Agreed. I'll incorporate this elsewhere.